### ISSUE TITLE
Doesn't work with fullcalendar 2.6.0

### ISSUE THREAD

**Issue by gawrrell** [mherrmann/fullcalendar-rightclick#6]
When I try to right click on a day I get this error:
TypeError: that.coordMap is undefined
that.coordMap.build();

**Comment by tahaipek** [comment]
change lines like below:

```
that.prepareHits();
 var cell = that.queryHit(ev.pageX, ev.pageY);
```

[Update to support fullcalendar 2.5+](https://github.com/mherrmann/fullcalendar-rightclick/pull/5)

**Comment by mherrmann** [comment]
Thanks for helping out @tahaipek. This is now fixed in version 1.7.
